test/integration_test.js

```diff
@@ -99,3 +99,78 @@ function clearConnectorFolder() {
         console.warn(e);
     }
 }
+
+// Start e2e
+chromedriver.start(); // on port 9515
+process.on('exit', chromedriver.stop);
+
+describe('plotly database connector', function Spec() {
```

Comment:
@chriddyp or @jackparmer it seems we got rid of this integration test in #181 merge.. Wanted to make sure that was intended, since it is the only integration test we had.

Reply:
It's still there: https://github.com/plotly/plotly-database-connector/blob/54c92ddcf83e1f0e5bde80aa0529b88c23513ebb/test/integration_test.js#L20-L25